spring-cloud-stream-binder-kafka/src/test/java/org/springframework/cloud/stream/binder/kafka/KafkaBinderTests.java

```diff
@@ -1663,6 +1372,517 @@ public void testAnonymousGroup() throws Exception {
 		binding2.unbind();
 	}
 
+	@Test
+	@SuppressWarnings("unchecked")
```

Comment:
I assume these test methods were moved for some reason; it makes it hard to review what changed, though.

Reply:
There are two different types of tests in this suite, one that uses the `getBinder` from the `AbstractTestBinder` and another one with custom configuration properties. I simply moved the latter set to the bottom and cleaned up a few things.